DropDownButton: describe `template` (#6073)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDropDownButton/1 Configuration/template.md

---
---
##### shortDescription
Specifies a custom template for the base button in DropDownButton.

##### param(data): Object
The button's data.

##### field(data.icon): String
The button's icon.

##### field(data.text): String
The button's text.

##### param(contentElement): DxElement
#include common-ref-elementparam with { element: "button content" }

##### return: String | Element | jQuery

> A template name or container.

Reviewer: I'm not sure about this one. Seems like this is the return value, so I probably wouldn't use "a" as an article. You would say "a template name" for a parameter... You can pass any template name. But if it's the return value, then this should be the result of some calculation or some query... So I'd likely say "the". Maybe something like this: 

```suggestion
The template name or the template's container element.
```

---

> The following code sample creates a DropDownButton featuring multiline text for the base button.

Reviewer: ```suggestion
The following code sample creates a DropDownButton with multiline text in the base button.
```
